lang/python/setup_pip.py

```diff
@@ -329,31 +261,37 @@ def source_filter(sources):
         d = os.path.join(stage_dir, os.path.dirname(f))
         if not os.path.isdir(d):
             os.makedirs(d)
-        if f in movers:
-            src = movers[f]
-        else:
-            src = f
         # Symlinks are not followed in setup, we need to use real files.
-        shutil.copy2(src, os.path.join(stage_dir, f))
+        shutil.copy2(f, os.path.join(stage_dir, f))
+    # Copy files in lang/python/wiredtiger to the root folder.
+    pywt_path = os.path.join(stage_dir, "lang", "python", "wiredtiger")
```

Comment:
copytree can only work for us from python 3.8+. Revert back to the old way of copying directory.